content/en/serverless/step_functions/installation.md

```diff
@@ -139,16 +139,27 @@ For developers using [Serverless Framework][4] to deploy serverless applications
 <div class="alert alert-warning"> If you are using a different instrumentation method such as Serverless Framework or datadog-ci, enabling autosubscription may create duplicated logs. Choose one configuration method to avoid this behavior.</div>
 
 4. Set up tags. Open your AWS console and go to your Step Functions state machine. Open the *Tags* section and add `env:<ENV_NAME>`, `service:<SERVICE_NAME>`, and `version:<VERSION>` tags. The `env` tag is required to see traces in Datadog, and it defaults to `dev`. The `service` tag defaults to the state machine's name. The `version` tag defaults to `1.0`.
-5. For Node.js and Python runtimes, you can link your Step Function traces to Lambda traces. On the Lambda Task, set the `Parameters` key with the following: 
+5. Link your Step Function traces to Lambda traces or other Step Functions:
```

Comment:
@kimi-p Should we mark this step as optional?

Reply:
Also, I'd change "other Step Functions" to "nested Step Functions"

Reply:
@kimi-p To clarify my dumb question: is there a case when the user doesn't want to merge Step Function trace with downstream Lambda/SFN trace? If it's always preferred to merge them, then I'm okay with the current code in the PR.

Reply:
1. `nested Step Functions` make sense to me!
2. Most customers want to link their `downstream Lambda` or nested step functions. So I think it's quite self-explanatory. But I would like to specify `downstream Lambda` explicitly because some customers do use Lambda to trigger Step Functions, which case we do not support right now.